### PR TITLE
[IMP] account_analytic_business_unit: Added related field business_unit_id

### PR DESCRIPTION
**Pull request by EmilioSerna**

### CODE REVIEW

**Comment by EmilioSerna** [comment]
Hi @hugho-ad could you please review it?

cc @angelesgl17

**Comment by angelesgl17** [comment]
@nhomar could u please review this? 

thanks in advance!

File: account_analytic_business_unit/models/account_move_line.py [hunk 1]
```diff
@@ -5,6 +5,9 @@ class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     analytic_line_count = fields.Integer(compute="_compute_total_analytic_lines")
+    business_unit_id = fields.Many2one(
+        related="analytic_account_id.business_unit_id", store=True, inverse="_inverse_analytic_account_id"
```
Comment (hugho-ad) [inline account_analytic_business_unit/models/account_move_line.py]: why the invserse?

Comment (hugho-ad) [reply account_analytic_business_unit/models/account_move_line.py]: pls add a help string

Comment (hugho-ad) [reply account_analytic_business_unit/models/account_move_line.py]: if you added it to allow edit it  then  ensure the record related don't change when modifying this

**Comment by EmilioSerna** [comment]
@hugho-ad Changes done, please review

File: account_analytic_business_unit/models/account_move_line.py [hunk 2]
```diff
@@ -5,6 +5,11 @@ class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     analytic_line_count = fields.Integer(compute="_compute_total_analytic_lines")
+    business_unit_id = fields.Many2one(
+        related="analytic_account_id.business_unit_id",
+        readonly=True,
```
Comment (hugho-ad) [inline account_analytic_business_unit/models/account_move_line.py]: related fields are  read-only
 by default

https://github.com/odoo/odoo/blob/c03172676a799cabda68e6927629d14d33b9aaa9/odoo/fields.py#L386

Comment (EmilioSerna) [reply account_analytic_business_unit/models/account_move_line.py]: Change applied, thanks

**Review by hugho-ad — APPROVED** [review]

**Pull request merged**